lib/matplotlib/table.py

```diff
@@ -838,5 +838,8 @@ def table(ax,
         if rowLabelWidth == 0:
             table.auto_set_column_width(-1)
 
+    # set_fontsize is only effective after cells are added
+    if fontsize is not None:
+        table.set_fontsize(fontsize)
```

Comment:
Add a blank line after this.  It will make it clear to the reader that adding the table is a separate step.